pyexchange/erisx.py

```diff
@@ -72,7 +72,7 @@ class ErisxApi(PyexAPI):
 
     def __init__(self, fix_trading_endpoint: str, fix_trading_user: str,
                  fix_marketdata_endpoint: str, fix_marketdata_user: str, password: str,
-                 clearing_url: str, api_key: str, api_secret: str, account_id: int = 0):
+                 clearing_url: str, api_key: str, api_secret: str, certs: dict, account_id: int = 0):
```

Comment:
nit: `ErisxFix` class defaults the param to `None`; curious why that isn't done here.

Reply:
No good reason, updated accordingly.